### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by skylorbeck** [lucaargolo/fabric-seasons#40]
```---- Minecraft Crash Report ----
// Daisy, daisy...

Time: 8/9/21, 10:36 PM
Description: Tesselating block model

java.lang.NullPointerException: Cannot invoke "java.util.Optional.orElseGet(java.util.function.Supplier)" because the return value of "net.minecraft.class_1959.redirect$zlp000$getSeasonGrassColor(net.minecraft.class_4763)" is null
	at net.minecraft.class_1959.method_8711(class_1959.java:407)
	at net.minecraft.class_638.method_23780(class_638.java:707)
	at net.minecraft.class_638.method_23781(class_638.java:689)
	at net.minecraft.class_4700.method_23770(class_4700.java:44)
	at net.minecraft.class_638.method_23752(class_638.java:689)
	at net.minecraft.class_853.method_23752(class_853.java:135)
	at net.minecraft.class_1163.method_4965(class_1163.java:14)
	at net.minecraft.class_1163.method_4962(class_1163.java:18)
	at net.minecraft.class_324.mdd3ad3e$lambda$create$1$9(class_324.java:1033)
	at net.minecraft.class_324.method_1697(class_324.java:112)
	at net.minecraft.class_778.method_23073(class_778.java:146)
	at net.minecraft.class_778.method_3363(class_778.java:132)
	at net.minecraft.class_778.method_3361(class_778.java:79)
	at net.minecraft.class_778.method_3374(class_778.java:48)
	at net.minecraft.class_776.method_3355(class_776.java:63)
	at net.minecraft.class_846$class_851$class_4578.redirect$zzf000$hookChunkBuildTesselate(class_846.java:727)
	at net.minecraft.class_846$class_851$class_4578.method_22785(class_846.java:524)
	at net.minecraft.class_846$class_851$class_4578.method_22783(class_846.java:447)
	at net.minecraft.class_846.mdd3ad3e$lambda$scheduleRunTasks$3$2(class_846.java:624)
	at java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1146)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1806)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1792)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:295)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1016)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1665)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1598)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```

https://pastebin.com/P7QGZNrP

**Comment by lucaargolo** [comment]
This issue should be fixed on the 1.3-BETA release.

**Comment by skylorbeck** [comment]
Thank you! Love the mod!
